scripts/explore_parameters.py

```diff
@@ -183,6 +187,8 @@ def step():
         t += 1
         if stepping: stepping = False
 
+agent = None
```

Comment:
this line is repeated on line 120. Do we want to delete it here?

Reply:
yes, resolved now.